subsys/bluetooth/controller/ll_sw/ull_sync_iso.c

```diff
@@ -11,29 +11,292 @@
 #include "common/log.h"
 #include "hal/debug.h"
 
+#include <sys/byteorder.h>
+
+#include "util/util.h"
+#include "util/mem.h"
+#include "util/memq.h"
+#include "util/mayfly.h"
+
+#include "hal/ccm.h"
+#include "hal/radio.h"
+#include "hal/ticker.h"
+
+#include "ticker/ticker.h"
+
+#include "pdu.h"
+#include "ll.h"
+
+#include "lll.h"
+#include "lll_vendor.h"
+#include "lll_clock.h"
+#include "lll_scan.h"
+#include "lll_sync.h"
+
+#include "ull_scan_types.h"
+#include "ull_sync_types.h"
+
+#include "ull_internal.h"
+#include "ull_scan_internal.h"
+#include "ull_sync_internal.h"
+#include "ull_sync_iso_internal.h"
+
+static struct ll_sync_iso ll_sync_iso_pool[CONFIG_BT_CTLR_SCAN_SYNC_ISO_MAX];
+static void *sync_iso_free;
+
+static int init_reset(void)
```

Comment:
Move the static functions after all non-static functions. Order of definitions:
1. HCI interface functions
2. Internal exported ULL functions restricted to ULL layer
3. Internal exported functions required by LLL layer (this should be avoided as much as possible to restrict circular dependencies between ULL and LLL layer)
4. File static functions (refactored out inline and reuse functions used by the above functions)
5. File static exported callback functions that would run asynchronously (order by first execution)

Reply:
Alright. One question: Why are we keeping the static functions at the bottom, rather than the top? Is it to somewhat enforce prototypes, or for some other reason?

Reply:
At this moment its just a style, to keep file public functions at top in the spec defined order of commands. This makes the file static functions be placed there after.